src/config/fenix.js

```diff
@@ -0,0 +1,127 @@
+import { transformAPIResponse } from '../utils/transform-data';
+import { extractBucketMetadata } from './shared';
+import { getProbeData } from '../state/api';
+import { validate, noResponse } from '../utils/data-validation';
+
+export default {
+  sampleRate: 1,
+  dimensions: {
+    os: {
+      title: 'OS',
+      key: 'os',
+      values: [{ key: 'Android', label: 'Android' }],
+    },
+    channel: {
+      title: 'Channel',
+      key: 'channel',
+      values: [
+        { key: 'fenixProduction', label: 'Firefox Preview' },
+        { key: 'release', label: 'Fenix' },
+      ],
+    },
+    pingType: {
+      title: 'Ping Type',
+      key: 'ping_type',
+      values: [
+        { key: '*', label: 'All' },
+        { key: 'metrics', label: 'Metrics' },
+        { key: 'baseline', label: 'Baseline' },
+      ],
+    },
+    aggregationLevel: {
+      title: 'Aggregation Level',
+      key: 'aggregationLevel',
+      values: [
+        { key: 'build_id', label: 'Build ID' },
+        { key: 'version', label: 'Major Version' },
+      ],
+    },
+  },
+  // FIXME: these are guesses at the moment
+  probeView: {
+    boolean: 'categorical',
+    custom_distribution_exponential: 'log',
+    custom_distribution_linear: 'log',
+    counter: 'linear',
+    timing_distribution: 'linear',
+    labeled_counter: 'categorical',
+    quantity: 'linear',
+  },
+  getParamsForQueryString(storeValue) {
+    // These parameters will map to a ${key}=${value}&... in the querystring,
+    // which is used to convey the view state when the GLAM URL is shared with
+    // others.
+    return {
+      channel: storeValue.productDimensions.channel,
+      os: storeValue.productDimensions.os,
+      pingType: storeValue.productDimensions.pingType,
+      aggregationLevel: storeValue.productDimensions.aggregationLevel,
+      timeHorizon: storeValue.timeHorizon,
+      proportionMetricType: storeValue.proportionMetricType,
+      activeBuckets: storeValue.activeBuckets,
+      visiblePercentiles: storeValue.visiblePercentiles,
+      reference: storeValue.reference,
+    };
+  },
+  getParamsForDataAPI(storeValue) {
+    // These parameters are needed to request the data from the API itself
+    return {
+      product: 'fenix',
+      channel: storeValue.productDimensions.channel,
+      os: storeValue.productDimensions.os,
+      pingType: storeValue.productDimensions.pingType,
+      probe: storeValue.probeName,
+      aggregationLevel: storeValue.productDimensions.aggregationLevel,
+    };
+  },
```

Comment:
Down the line, can/should this be generalized? These lines are almost identical to those in *firefox-desktop.js*.

Reply:
I now see that you mention this below.